etc/image-input.conf

```diff
@@ -0,0 +1,86 @@
+#
+# You can use modifier-key combinations like Shift+Left or Ctrl+Alt+x with
+# the modifiers Shift, Ctrl, Alt and Meta (may not work on the terminal).
+#
+# The default keybindings are hardcoded into the mpv binary.
+# You can disable them completely with: --no-input-default-bindings
+
+#LEFT        script-binding/positioning pan-x -0.2 # pan left
+#DOWN        script-binding/positioning pan-y  0.2 # pan down
+#UP          script-binding/positioning pan-y -0.2 # pan up
+#RIGHT       script-binding/positioning pan-x  0.2 # pan right
+#h           script-binding/positioning pan-x -0.2 # pan left
+#j           script-binding/positioning pan-y  0.2 # pan down
+#k           script-binding/positioning pan-y -0.2 # pan up
+#l           script-binding/positioning pan-x  0.2 # pan right
+#Shift+LEFT  script-binding/positioning pan-x -0.02 # pan left slowly
+#Shift+DOWN  script-binding/positioning pan-y  0.02 # pan down slowly
+#Shift+UP    script-binding/positioning pan-y -0.02 # pan up slowly
+#Shift+RIGHT script-binding/positioning pan-x  0.02 # pan right slowly
+#H           script-binding/positioning pan-x -0.02 # pan left slowly
+#J           script-binding/positioning pan-y  0.02 # pan down slowly
+#K           script-binding/positioning pan-y -0.02 # pan up slowly
+#L           script-binding/positioning pan-x  0.02 # pan right slowly
+
+# Recommened on a touchpad:
+# WHEEL_UP    script-binding/positioning pan-y -0.2 # pan up
+# WHEEL_DOWN  script-binding/positioning pan-y  0.2 # pan down
+# WHEEL_LEFT  script-binding/positioning pan-x -0.2 # pan left
+# WHEEL_RIGHT script-binding/positioning pan-x  0.2 # pan right
+
+#Ctrl+LEFT   no-osd set video-align-x -1 # align to the left
+#Ctrl+DOWN   no-osd set video-align-y  1 # align to the bottom
+#Ctrl+UP     no-osd set video-align-y -1 # align to the top
+#Ctrl+RIGHT  no-osd set video-align-x  1 # align to the right
+#Ctrl+h      no-osd set video-align-x -1 # align to the left
+#Ctrl+j      no-osd set video-align-y  1 # align to the bottom
+#Ctrl+k      no-osd set video-align-y -1 # align to the top
+#Ctrl+l      no-osd set video-align-x  1 # align to the right
+
+= add video-zoom  0.25 # zoom in
+- add video-zoom -0.25 # zoom out
++ add video-zoom  0.05 # zoom in slowly
+_ add video-zoom -0.05 # zoom out slowly
+0 no-osd set video-zoom 0; no-osd set panscan 0 # reset zoom
+
+#u no-osd cycle-values video-unscaled yes no; no-osd set video-zoom 0; no-osd set panscan 0 # toggle scaling the image to the window.
+
+#o no-osd set panscan 1; no-osd set video-unscaled no; no-osd set video-zoom 0 # fill black bars
+
+#MBTN_RIGHT script-binding positioning/drag-to-pan # pan around the clicked point
+
+#WHEEL_UP   script-message cursor-centric-zoom  0.1 # zoom in towards the cursor
+#WHEEL_DOWN script-message cursor-centric-zoom -0.1 # zoom out towards the cursor
+
+#s cycle-values image-display-duration 5 inf; no-osd set video-zoom 0; no-osd set panscan 0; no-osd set video-unscaled no # toggle slideshow
```

Comment:
Because if you're zoomed in and enable the slideshow, it is assumed that you want to keep viewing images without interaction and thus want to display the entire image fit to the window without having to change the zoom. But it can be removed, if you think it's too opinionated.